be2-scala/src/main/scala/ch/epfl/pop/pubsub/graph/validators/MessageDataContentValidator.scala

```diff
@@ -58,6 +58,15 @@ trait MessageDataContentValidator extends ContentValidator with AskPatternConsta
       Right(error)
   }
 
+  /** This method behaves the same as checkTimestampOrder, except that the param <second> is not necessarily defined. (The protocol states that <data.en> is not necessarily defined for certain type of messages as CreateMeeting)
```

Comment:
`(Wrt to the protocol, some fields are not necessarily defined for certain type of messages, such as CreateMeeting)`